### ISSUE TITLE
[artifactory-ha] Using gcpServiceAccount for googleStorage throws error

### ISSUE THREAD

**Issue by dhdersch** [jfrog/charts#805]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): BUG REPORT

**Version of Helm and Kubernetes**:
Kubernetes: 

```
Server Version: version.Info{Major:"1", Minor:"15+", GitVersion:"v1.15.11-gke.3", GitCommit:"71631703d37fb1b4b333abd427deae839aeb2032", GitTreeState:"clean", BuildDate:"2020-03-24T18:54:48Z", GoVersion:"go1.12.17b4", Compiler:"gc", Platform:"linux/amd64"}
```

Helm:

```
version.BuildInfo{Version:"v3.1.2", GitCommit:"d878d4d45863e42fd5cff6743294a11d28a9abce", GitTreeState:"clean", GoVersion:"go1.13.8"}
```

**Which chart**:
jfrog/artifactory-ha
2.3.1

**What happened**:
I attempted to deploy the chart with `artifactory.persistence.googleStorage.gcpServiceAccount.enabled=true` specified. I also tried specifying the config in the values.yaml and by using a secret as described in the instructions. In both cases, I get the following error:

```
Error: template: artifactory-ha/templates/artifactory-primary-statefulset.yaml:38:36: executing "artifactory-ha/templates/artifactory-primary-statefulset.yaml" at <include (print $.Template.BasePath "/artifactory-gcp-credentials.yaml") .>: error calling include: template: no template "artifactory-ha/templates/artifactory-gcp-credentials.yaml" associated with template "gotpl"
```

**What you expected to happen**: I expected artifactory-ha to be deployed successfully using a google cloud service account.

**How to reproduce it** (as minimally and precisely as possible): All you have to do is try to use a gcpServiceAccount and the issue will occur.

**Anything else we need to know**:

**Comment by chukka** [comment]
@dhdersch Thanks for the bug report. let me investigate further and get back to you soon

**Comment by stickyPins** [comment]
Thanks Chukka for looking into this. Any updates yet?

**Comment by chukka** [comment]
@stickyPins @dhdersch Looks like a bug . We are working on a fix and update you soon

**Comment by chukka** [comment]
Fixed as part of 2.4.1 chart version- please try and provide feedback :)
